pythonforandroid/archs.py

```diff
@@ -149,20 +127,27 @@ def get_env(self, with_flags_in_cc=True, clang=False):
                 execxx=execxx,
                 ccache=ccache)
 
+        # Android's binaries
+        command_prefix = self.command_prefix
         env['AR'] = '{}-ar'.format(command_prefix)
         env['RANLIB'] = '{}-ranlib'.format(command_prefix)
-        env['LD'] = '{}-ld'.format(command_prefix)
-        env['LDSHARED'] = env["CC"] + " -pthread -shared " +\
-            "-Wl,-O1 -Wl,-Bsymbolic-functions "
-        if self.ctx.python_recipe and self.ctx.python_recipe.from_crystax:
-            # For crystax python, we can't use the host python headers:
-            env["CFLAGS"] += ' -I{}/sources/python/{}/include/python/'.\
-                format(self.ctx.ndk_dir, self.ctx.python_recipe.version[0:3])
         env['STRIP'] = '{}-strip --strip-unneeded'.format(command_prefix)
-        env['MAKE'] = 'make -j5'
+        env['MAKE'] = 'make -j{}'.format(str(cpu_count()))
```

Comment:
@KeyWeeUsr, I was thinking with you when I changed this line :wink:, we still have pending work related to this (#1558)...

**Note:** I put this in here as a reminder